### PR TITLE
make ext-ffi suggested 

### PR DESCRIPTION
**Pull request by MrBalum**
This PR should make the ffi extension suggested in the composer, this is necessesary because projects that resolve this library as dependency will break if they want to implement this library as suppressable library

### CODE REVIEW

**Comment by MrBalum** [comment]
We delete the .lock file to prevent predefining dependency versions

File: composer.json [hunk 1]
```diff
@@ -1,15 +1,17 @@
...
+    "psr-4": {
+      "DexproSolutionsGmbh\\SoftekBarcodeWrapper\\": "src/"
     }
+  },
+  "require": {
+    "php": ">=7.4"
+  },
+  "suggest": {
+    "ext-ffi": "*"
+  }
```
Comment (FabianTe) [inline composer.json:16]: I think we should document in the README that FFI is a mandatory dependency for this package. It should also mention why we list FFI as suggested dependency.

Example:

```md

## Requirements

- `ext-ffi` - This package requires FFI. It is listed as suggested dependency .......

```

**Review by FabianTe — CHANGES_REQUESTED** [review]
Change is fine to me but we should document why FFI is a suggested dep.

File: composer.json (hunk 1 above)
Comment (MrBalum) [reply composer.json:16]: done

**Review by FabianTe — APPROVED** [review]

**Pull request merged**